test/functional/inspec_init_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'functional/helper'
+require 'fileutils'
+require 'tmpdir'
+require 'yaml'
+
+describe 'inspec init' do
+  include FunctionalHelper
+
+  tmpdir = Dir.tmpdir
+  puts tmpdir
+
+  describe 'inspec init profile with/slash' do
+    it 'Names profile with string after last slash' do
```

Comment:
Usually we use lower case here i.e: `it 'names....` but I won't lose sleep if you don't change it :smile: